cmd/aws-snapshot/main.go

```diff
@@ -75,6 +75,7 @@ func realMain() error {
 		timeout = &t
 	}
 
+	stdr.SetVerbosity(6)
```

Comment:
The issue with hardcoding this here is that it may mislead users to think that a given message is logged by default, when in fact they must increase their verbosity in order to see the same result.

Add a `-v` flag which is parsed from the command line that sets this verbosity, otherwise leave as is.